### PR TITLE
WIP: Initial commit for upgrades

### PR DESCRIPTION
**Pull request by andytael**
Upgrades to resolve some CVEs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,22 +16,21 @@ For a deep dive into the project, refer to the Spring Cloud OCI documentation:
...
 
 This project has dependency and transitive dependencies on Spring Projects. The table below outlines the versions of Spring Cloud, Spring Boot and Spring Framework versions that are compatible with certain Spring Cloud OCI version.
 
-| Spring Cloud OCI  | Spring Cloud              | Spring Boot         | OCI Java SDK |
-|-------------------|---------------------------|---------------------|--------------|
-| 1.0.0  	        | 2022.0.x                  | 3.1.x, 3.0.x        | 3.24.0       |
-
+| Spring Cloud OCI | Spring Cloud       | Spring Boot         | OCI Java SDK |
+|------------------|--------------------|---------------------|--------------|
+| 1.0.0           | 2022.0.x, 2023.0.x | 3.2.x, 3.1.x, 3.0.x | 3.41.1       |
```
Comment (markxnelson) [inline README.md:25]: i think we should rev it, leave 1.0.0 as it was, and add a 1.1.0 or 1.0.1 with the new stuff

File: pom.xml [hunk 2]
```diff
@@ -183,6 +183,29 @@ Licensed under the Universal Permissive License v 1.0 as shown at https://oss.or
         </pluginManagement>
 
 		<plugins>
+            <!-- OpenRewrite Plugin
```
Comment (markxnelson) [inline pom.xml:186]: eep

Comment (markxnelson) [reply pom.xml:186]: remove this?

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andytael) [reply README.md]: Agreed, added a 1.0.1

File: pom.xml (hunk 2 above)
Comment (andytael) [reply pom.xml]: Gone with the wind

**Review by markxnelson — APPROVED** [review]

**Pull request merged**